test/unit/models/admin/activities/builder.test.js

```diff
@@ -0,0 +1,13 @@
+const { ActivityAddedViewModel } = require('../../../../../app/models/admin/activities/builder')
+describe('builder', () => {
+  describe('ActivityAddedViewModel', () => {
+    test('should create receive models', () => {
+      const activityAddedViewModel = ActivityAddedViewModel({
+        label: 'Judicial review notice',
+        activitySource: 'Dog',
+        activityType: 'receive'
+      })
+      expect(activityAddedViewModel.model.bodyContent).toEqual(['Judicial review notice is available in the Dog record receive activities.'])
```

Comment:
We should probably enhance these tests to check more than just bodyContent, but the other properties too (as for the other model tests below). Another one for later.